test/db/cmd/cmd_tc

```diff
@@ -168,8 +166,6 @@ FILE=bins/mach0/objc-employee-ios14-arm64
 CMDS=ic*
 EXPECT=<<EOF
 fs classes
-"f class.public.Employee @ 0x100007958"
```

Comment:
Why this disappears?

Reply:
because before there was a flag for each `addr`, now since we remove it, we don't anymore.